trio/_core/_tests/test_ki.py

```diff
@@ -4,7 +4,7 @@
 import inspect
 import signal
 import threading
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, AsyncIterator, Callable, Iterator
```

Comment:
I would import from `collections.abc` instead

Reply:
going to leave that for some autofixer / I don't care enough to bother fixing it manually.